### DISCUSSION TITLE
Setting Rill locally

### DISCUSSION THREAD

**Discussion by cloudcompute** [rilldata/rill#5181] [Q&A]
Hi

I want to setup Rill locally (both web and backend). is there any script for it?

I am seeing Makefile but it creates a CLI executable.

**Comment by AndrewRTsao** [comment]
Hey cloudcompute, could you share more details about your use case? We actually provide an [install script](https://docs.rilldata.com/home/install) that will set up Rill Developer on your local machine. Then, for dashboards that you wish to share with others or consume in a production capacity, the recommended path is to actually deploy the [project to Rill Cloud](https://docs.rilldata.com/deploy/existing-project/).

If you had something else in mind or a different use case, we'd love to learn more about it and see what we can do to help.

**Comment by cloudcompute** [comment]
Hi @AndrewRTsao 

I am aware of this script. Actually, it takes me to Rill cloud, I want to set up the environment locally (including Org create) and probably add more connectors to it (if any of my customer asks for it). I am a developer and therefore interested in local setup, that is all.

**Reply by AndrewRTsao** [reply]
Hey @cloudcompute unfortunately providing a self-hosted Rill Cloud offering is not on our immediate roadmap. If this is something that you wish to continue to pursue after building and deploying a few dashboards, there may be a few options that we can consider but this would involve having an upfront commercial conversation. Let me know how you'd like to proceed!

**Comment by cloudcompute** [comment]
Hi @AndrewRTsao 

If I have a customer asking for it, I will let you know. 

Thanks
